ops/testing.py

```diff
@@ -1761,15 +1761,9 @@ def start_services(
             if name not in known_services:
                 # TODO: jam 2021-04-20 This needs a better error type
                 raise RuntimeError('400 Bad Request: service "{}" does not exist'.format(name))
-            current = self._service_status.get(name, pebble.ServiceStatus.INACTIVE)
-            if current == pebble.ServiceStatus.ACTIVE:
-                # TODO: jam 2021-04-20 I believe pebble actually validates all the service names
-                #  can be started before starting any, and gives a list of things that couldn't
-                #  be done, but this is good enough for now
-                raise pebble.ChangeError('''\
-cannot perform the following tasks:
-- Start service "{}" (service "{}" was previously started)
-'''.format(name, name), change=1234)  # type:ignore # the change id is not respected
+            # TODO: jam 2021-04-20 I believe pebble actually validates all the service names
+            #  can be started before starting any, and gives a list of things that couldn't
+            #  be done, but this is good enough for now
         for name in services:
             # If you try to start a service which is started, you get a ChangeError:
```

Comment:
This comment (including the next three lines) are no longer true, so this comment block should be removed.

Reply:
While you're here, might also be good to remove the comment "TODO: handle invalid names" below (line 1790), as we do actually check for unknown service names.

Reply:
Oh also, you'll need to merge the latest changes from the main branch to bring this up to date.

Reply:
Sure! Thank you for the review! I hope you had great holidays over the past two weeks :)
Let me work the comments mentioned above and push right away!